scene/gui/tree.cpp

```diff
@@ -4215,10 +4215,22 @@ Tree::SelectMode Tree::get_select_mode() const {
 }
 
 void Tree::deselect_all() {
+	if (root) {
```

Comment:
Would be nice to use `root` or `get_root()` consistently inside of the same method.